app/views/schools/choose_programme/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+  <h1 class="govuk-heading-xl">Welcome</h1>
+  <%= form_for @induction_choice_form, url: schools_choose_programme_path, method: :post do |f| %>
+    <%= f.govuk_collection_radio_buttons(
+          :programme_choice, @induction_choice_form.programme_choices, :id, :name,
+          legend: { text: "How do you want to run your induction in the 2021/22 academic year?", size: "m" },
+          hint: -> { tag.p do
+            concat(govuk_link_to("See our guidance on programmes", "https://www.gov.uk/government/publications/early-career-framework-reforms-overview/early-career-framework-reforms-overview", target: "_blank", rel: "noopener noreferrer"))
```

Comment:
`early-career-framework-reforms-overview` appears twice in the url - is this intentional?

Reply:
Yep. As for why... ask comms 😛